proxy/proxy.go

```diff
@@ -515,13 +547,26 @@ func (p *Proxy) do(ctx *context) error {
 		ctx.outgoingDebugRequest = debugReq
 		ctx.setResponse(&http.Response{Header: make(http.Header)}, p.flags.PreserveOriginal())
 	} else {
+		done, allow := p.checkBreaker(ctx)
+		if !allow {
+			return errCircuitBreakerOpen
```

Comment:
valid point. It should be there.

Besides the logs, the plan is to add an endpoint to the "metrics" -> "support" listener that reports all the breaker states. After https://github.com/zalando/skipper/pull/364 gets finished.